### ISSUE TITLE
Wrong changelog generated when having objects with same field name

### ISSUE THREAD

**Issue by gbalabanov** [OpenAPITools/openapi-diff#268]
Lets say that the old and the new OpenAPI schemas have classes **Object1** and **Object2**, both of which have a field _myField1_ without changes in-between the openapi schemas. If **Obejct1.myField1** has different description than **Object2.myField1**, that would be detected as a change and printed in the output diff file.

Also, can you add some kind of configuration that would prevent the library to print such non-breaking changes (i.e. changed description is not a breaking change) ?

**Comment by joschi** [comment]
@gbalabanov Could you please provide a test case (minimal OpenAPI spec) to demonstrate the issue?

**Comment by gbalabanov** [comment]
Please find attached

**Comment by joschi** [comment]
@gbalabanov I think you forgot to attach the specs. 😉

**Comment by gbalabanov** [comment]
Yes, json and md files are not supported as attachments. Changed all of the to txt.
[test2.txt](https://github.com/OpenAPITools/openapi-diff/files/7294606/test2.txt)
[diff.txt](https://github.com/OpenAPITools/openapi-diff/files/7294607/diff.txt)
[test1.txt](https://github.com/OpenAPITools/openapi-diff/files/7294608/test1.txt)
